.gitignore

```diff
@@ -56,3 +56,4 @@ storybook-static
 
 # Non-Yarn lockfiles
 package-lock.json
+.vscode
```

Comment:
Nit: should we create another "section" in here (via a comment - `#`) for editor config files?